packages/web-runtime/src/components/SidebarQuota.vue

```diff
@@ -22,7 +22,7 @@ export default {
     ...mapGetters(['quota']),
 
     usedQuota() {
-      return getResourceSize(this.quota.used)
+      return getResourceSize(this.quota.used) || '0 B'
```

Comment:
yes you are right. My thought was that the same function is also defined in https://github.com/owncloud/web/blob/14acebbf3948bfcad86e0e3389143549c38c5e89/packages/web-app-files/src/mixins/resources.js#L10 and used in https://github.com/owncloud/web/blob/f46085ad7dba7cfc7d35a953c796714e0e88b4e1/packages/web-app-files/src/components/InfoSelectedResources.vue
so in the next step I wanted to consolidate both functions (not sure where it has to be placed to be used from both though)
Now looking at `InfoSelectedResources.vue` in detail I think returning `0` instead of `''` would not harm there ether

Reply:
We introduced `packages/web-pkg` a while ago for shared code, I think it would be good to have it there.